templates/greenlight/greenlight-redirect.nginx.j2

```diff
@@ -0,0 +1,3 @@
+location = / {
```

Comment:
Please also make this optional (based on the `bbb_greenlight_enable` variable, as before)

Reply:
it is optional by:
https://github.com/n0emis/ansible-role-bigbluebutton/blob/6b106944df8a09d3e8d773f2b5c8b19457205df3/tasks/main.yml#L136-L137

when this file is copied here:
https://github.com/n0emis/ansible-role-bigbluebutton/pull/13/files#diff-4a0a691a12b85e5f74cea98ea19411ceR43-R49

Reply:
:facepalm: of course...